ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ptp.schema.yml

```diff
@@ -14,6 +14,8 @@ keys:
         valid_values:
         - boundary
         - transparent
+      boundary_one_step:
```

Comment:
I am not convinced this is teh best model when looking at my CLI:

```
l3leaf1(config)#show cli commands | grep "ptp mode"
[no|default] ptp mode MODE
ptp mode boundary one-step
ptp mode e2etransparent one-step
```

Looks like at least one other mode that we don't support today supports `one-step`
so I would rather we call this `one_step` and document for which modes it is supported

@ClausHolbechArista

Reply:
Right, like mode_one_step as a boolean.

Reply:
one-step is for these two only, right?
shall I add conditions for both ?

Reply:
From what I can see in my CLI in Cloud EOS yes - so let's document it like this for now.

Use the `mode_one_step` as Claus suggested! Thanks!